tests/algorithms/sbi/test_abc_run_methods.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+import torch
+
+import sbibm
+from sbibm.algorithms.sbi.mcabc import run as run_mcabc
+from sbibm.algorithms.sbi.smcabc import run as run_smcabc
+from sbibm.metrics.c2st import c2st
+
+
+@pytest.mark.parametrize(
+    "task_name",
+    [tn for tn in ["gaussian_linear", "two_moons"]],
```

Comment:
maybe shorten this with`("gaussian_linear", "two_moons")` instead of the list comprehension?

Reply:
:facepalm: thanks for spotting this. 1800192